pwndbg/commands/nearpc.py

```diff
@@ -80,7 +80,23 @@ def nearpc(pc=None, lines=None, to_string=False, emulate=False):
     #             pc_to_linenos[line.pc].append(line.line)
     instructions = pwndbg.disasm.near(pc, lines, emulate=emulate)
```

Comment:
I would much rather this logic be implemented in `pwndbg.disasm.near`, rather than in the command itself.